pkgs/development/python-modules/pillow/default.nix

```diff
@@ -22,6 +24,51 @@ import ./generic.nix (rec {
     sha256 = "f401ed2bbb155e1ade150ccc63db1a4f6c1909d3d378f7d1235a44e90d75fb97";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "CVE-2022-45198.patch";
+      url = "https://github.com/python-pillow/Pillow/commit/c9f1b35e981075110a23487a8d4a6cbb59a588ea.patch";
+      hash = "sha256-jaITGw3bc+ntcYgt8NG9H4cgDWCqYKFKqkL4SeqRB6w=";
+    })
+    # this is only the test-case added from the CVE-2022-45199, as
+    # a means to "prove" that 9.1.0 isn't vulnerable
+    (fetchpatch {
+      name = "CVE-2022-45199-test.patch";
+      url = "https://github.com/python-pillow/Pillow/commit/2444cddab2f83f28687c7c20871574acbb6dbcf3.patch";
+      excludes = [
+        "docs/releasenotes/9.3.0.rst"
+        # the "fix"
+        "src/PIL/TiffImagePlugin.py"
+      ];
+      hash = "sha256-P63rLbY2AOEXVDcOCUzwHRH8JmgieAIfGgiXPK7P4O0=";
+    })
+  ];
+
+  # patching mechanism doesn't work with binary files, but the commits contain
+  # example images needed for the accompanying tests, so invent our own mechanism
+  # to put these in place
```

Comment:
@mweinelt it's baaack...

Reply:
How about we just skip those tests?

Reply:
The test is what proves the patch has had the desired effect.

Reply:
cc #204320